packages/sdk-react/src/components/approvalPolicies/ApprovalPoliciesTable/ApprovalPoliciesTable.tsx

```diff
@@ -133,101 +133,104 @@ const ApprovalPoliciesTableBase = ({
-              py: '22px',
-            },
-            '& .MuiDataGrid-withBorderColor': {
-              borderColor: 'divider',
-            },
-            '&.MuiDataGrid-withBorderColor': {
-              borderColor: 'divider',
-            },
-          }}
-          loading={isLoading}
-          getRowHeight={() => 'auto'}
-          columns={[
-            {
-              field: 'name',
-              headerName: t(i18n)`Policy name`,
-              sortable: false,
-              flex: 1,
-            },
-            {
-              field: 'triggers',
-              headerName: t(i18n)`Triggers`,
-              sortable: false,
-              flex: 1,
-              renderCell: (params) => (
-                <ApprovalPoliciesTriggers approvalPolicyId={params.row.id} />
-              ),
-            },
-            {
-              field: 'rule',
-              headerName: t(i18n)`Rule`,
-              sortable: false,
-              flex: 1,
-              renderCell: (params) => (
-                <ApprovalPoliciesRules approvalPolicyId={params.row.id} />
-              ),
-            },
-            {
-              field: 'created_at',
-              headerName: t(i18n)`Created at`,
-              sortable: false,
-              flex: 0.7,
-              valueFormatter: (value) =>
-                i18n.date(value, DateTimeFormatOptions.EightDigitDate),
-            },
-            {
-              field: 'created_by',
-              headerName: t(i18n)`Created by`,
-              sortable: false,
-              flex: 0.8,
-              renderCell: ({ value }) => (
-                <ApprovalPoliciesUser entityUserId={value} />
-              ),
-            },
-          ]}
-          rows={approvalPolicies?.data || []}
-          onRowClick={(params) => onRowClick?.(params.row)}
-          slots={{
-            pagination: () => (
-              <TablePagination
-                nextPage={approvalPolicies?.next_pagination_token}
-                prevPage={approvalPolicies?.prev_pagination_token}
-                paginationModel={{
-                  pageSize,
-                  page: currentPaginationToken,
-                }}
-                onPaginationModelChange={({ page, pageSize }) => {
-                  setPageSize(pageSize);
-                  setCurrentPaginationToken(page);
-                }}
-              />
-            ),
-          }}
-        />
+    <Box
+      className={ScopedCssBaselineContainerClassName}
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        overflow: 'hidden',
+        height: 'inherit',
+        pt: 2,
+      }}
+    >
+      <Box sx={{ mb: 2 }}>
+        <Filters onChangeFilter={onChangeFilter} />
       </Box>
-    </>
+      <DataGrid
+        disableColumnFilter={true}
+        rowSelection={false}
+        sx={{
+          '&.MuiDataGrid-root--densityCompact .MuiDataGrid-cell': {
+            py: '8px',
```

Comment:
In my opinion, it would be better to use something like `py: 2` to keep internal spacing consistent.

Reply:
@radist2s I agree. Selector '&.MuiDataGrid-root--densityCompact .MuiDataGrid-cell' itself isn't great though. I wouldn't worry about hardcoded 8px until this selector moved to a theme file.
This refactoring is out of scope of this PR though.